docs/react-v9/contributing/new-components.md

```diff
@@ -0,0 +1,87 @@
+# Contributing a new component to Fluent UI React
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31611&commitId=6e95f242340c34d54f708a54d12e2921149c3972&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Added|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Added|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Added|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.CardHeader.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default.default.chromium.png|0| Added|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - RTL.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>